documentation/release-notes/Stack/migration-guide.1.2.11.md

```diff
@@ -0,0 +1,251 @@
+# Table of Contents
+1. [Removal of Force parameters](#removal-of-force-parameters)
+2. [Change of Tag parameters](#change-of-tag-parameters)
+3. [Breaking Changes to Profile Cmdlets](#breaking-changes-to-profile-cmdlets)
```

Comment:
We also have changes in the Get-AzureRmSubscription and Select-AzureRmSubscription cmdlets differring in outputs from 1.2.10

We cannot pipe azurestack subscription model to Select-AzureRmSubscription, we need to call these out